apollo-router/src/plugins/authentication/mod.rs

```diff
@@ -403,6 +403,16 @@ fn authenticate(
             monotonic_counter.apollo_authentication_failure_count = 1u64,
             kind = %AUTHENTICATION_KIND
         );
+        tracing::info!(
+            monotonic_counter
+                .apollo
```

Comment:
metric name is wrong

Reply:
Fixed in [2def84a](https://github.com/apollographql/router/pull/3354/commits/2def84a927cc20ccec6fd0d9ba94d9b183676b78)